CONTRIBUTING.md

```diff
@@ -326,10 +326,11 @@ the `find-bug.py` script in that repo).
 
 ### Beta testing ###
 
-Coq gets a new major release about every six months.  Before a new
-major version is released, there is a beta-testing period, which
-usually lasts one month (see the [release plan][release-plan]).  You
-can help make the upcoming release better, by testing the beta
+Coq gets a new major release about every six months, which is then
+distributed through the [Coq Platform][Coq-Platform].
+New major versions of Coq are first made available for beta-testing,
+before being declared stable and the new default version of the Coq
+Platform. You can help make by testing the beta
```

Comment:
No mention of opam?

Reply:
What kind of mention would you expect / like to see here?

Reply:
Well reading this the first time, it seems that it is only distributed through the Coq platform, which of course isn't true. However I suppose it isn't that important to mention here really.